src/components/image.html

```diff
@@ -11,7 +11,7 @@
       crossorigin="anonymous"
     />
     <link rel="stylesheet" href="../../css/style.css" />
-
+    <script src="../../js/script.js"></script>
```

Comment:
I think script tag should be added at the end of the body tag not in head of HTML. It could stop the HTML from getting loaded quicker

Reply:
I have included only functions in the script file. So I didn't face any issue. Will update.